TEST/ar1.py

```diff
@@ -0,0 +1,3 @@
+import numpy as n
+a=n.array([1,2,3,4,5])
+print(a)

```

Comment:
WHAT ARE YOU GOING TO PRINT? CAN YOU PELASE EXPLORE ME?

Reply:
Here I'm going to print the value of a which stored the data in the form of array